frontend/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
```

Comment:
Any preference on versioning and if this should start differently than 0.1.0? 🤷‍♂

Reply:
I think 0.0.1 is most appropriate